qiskit/scheduler/methods/basic.py

```diff
@@ -121,69 +111,3 @@ def update_times(inst_qubits: List[int], time: int = 0) -> None:
     timed_schedules = [(time, cpd.schedule) for time, cpd in zip(start_times, circ_pulse_defs)
                        if not isinstance(cpd.schedule, Barrier)]
     return Schedule(*timed_schedules, name=circuit.name)
-
-
-def translate_gates_to_pulse_defs(circuit: QuantumCircuit,
```

Comment:
Should this method be deprecated since it's been renamed? Or is it just used locally in this file?

Reply:
It's only used here. Technically, it should be deprecated, but in this case I don't think it's necessary. I don't really consider it part of the user API